nntrainer/tensor/blas_neon.cpp

```diff
@@ -351,10 +413,8 @@ void sgemv_neon_fp16(const __fp16 *A, const __fp16 *X, __fp16 *Y, uint32_t rows,
       y = vfmaq_f16(y, wvec48_55, x48_55);
```

Comment:
These wvecX_Y variables can also be written inline as previous. Will improve performance.

Reply:
Fixed. Thanks!